README.md

```diff
@@ -25,21 +23,27 @@ npm install svelte-tag
 ```
 
 ```javascript
-import Component from "svelte-tag"
+import svelteTag from "svelte-tag"
```

Comment:
This was an opinionated change. In my defense, not only did I naturally want to name the function `svelteTag`, but apparently so have others (see dependents). Seems natural 😊

Reply:
e.g. https://github.com/souriya777/elegant-seo-website/blob/main/_dev/src/js/svelte-elements.js  (actively under development it appears, so here's a screenshot just in case it disappears):

![image](https://github.com/crisward/svelte-tag/assets/4269377/c2520644-12e1-4b3e-82bd-5be19571fe8a)